cos_alerter/config-defaults.yaml

```diff
@@ -9,12 +9,16 @@ watch:
   wait_for_first_connection: true
 
   # The list of Alertmanager instances we are monitoring. Alertmanager instances should be
-  # configured with the clientid=<client> parameter.
+  # configured with the clientid=<clientid> &key=<clientkey> parameters.
```

Comment:
This is where I would like to see the description of what should be in these values.